pypesto/objective/base.py

```diff
@@ -241,6 +241,8 @@ def check_sensi_orders(
             Boolean indicating whether combination of sensi_orders and mode
             is supported
         """
+        if(max(sensi_orders) > 2):
```

Comment:
if checking this, then better also for fun and res separately (the latter can only <=1)

Reply:
:+1: